### PR TITLE
Feature/36

### PR DESCRIPTION
**Pull request by sglover**

### CODE REVIEW

File: lmctl/cli/commands/lifecycledriver.py [hunk 1]
```diff
@@ -29,17 +30,24 @@ def format_lifecycle_driver(output_format, lifecycle_driver):
...
-        lifecycle_driver = lifecycle_mgmt_driver.add_lifecycle_driver(new_lifecycle_driver)
-    click.echo(format_lifecycle_driver(output_format, lifecycle_driver))
+    try:
+        lifecycle_mgmt_driver = get_lifecycle_driver_mgmt_driver(environment, config, pwd)
+        new_lifecycle_driver = {
+            'type': lifecycle_type,
+            'baseUri': url
+        }
+        if certificate is not None:
+            new_lifecycle_driver['certificate'] = read_certificate_file(certificate)
```
Comment (dvaccarosenna) [inline lmctl/cli/commands/lifecycledriver.py]: Could we just wrap this line in the "try/except"?

File: lmctl/utils/__init__.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (dvaccarosenna) [inline lmctl/utils/__init__.py]: Could this go in a "file" or "certificate" module of utils?

Comment (sglover) [reply lmctl/utils/__init__.py]: Yep

File: lmctl/cli/commands/lifecycledriver.py (hunk 1 above)
Comment (sglover) [reply lmctl/cli/commands/lifecycledriver.py]: ok

**Pull request merged**